cmd/s3-gw/app_settings.go

```diff
@@ -121,11 +122,17 @@ const ( // Settings.
 	// Number of the object copies to consider PUT to NeoFS successful.
 	cfgSetCopiesNumber = "neofs.set_copies_number"
 
+	//  Timeout between retrieving actual epoch from NeoFS. Actual only if slicer.enabled = true.
+	cfgEpochUpdateInterval = "neofs.epoch_update_interval"
+
 	// List of allowed AccessKeyID prefixes.
 	cfgAllowedAccessKeyIDPrefixes = "allowed_access_key_id_prefixes"
 
 	// envPrefix is an environment variables prefix used for configuration.
 	envPrefix = "S3_GW"
+
+	// Shows if slicer is enabled. If enabled slicer will be used for object put.
+	cfgSlicerEnabled = "slicer.enabled"
```

Comment:
`internal_slicer`? I'm not sure what would be the best name for it.

Reply:
I'm not sure, that I like _internal_ in this case

Reply:
Internal for the S3 gateway. Otherwise it happens in the NeoFS node, thus can be considered external to the S3 gateway. Maybe there are better options.